apps/events/views.py

```diff
@@ -0,0 +1,120 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import get_object_or_404, render
+from django.urls import reverse
+
+from apps.events.forms import EventActionForm, StaticTriggerForm, TimeoutTriggerForm
+from apps.events.models import StaticTrigger, TimeoutTrigger
+from apps.utils.time import seconds_to_human
+
+
+def create_timeout_event_view(request, team_slug: str, experiment_id: str):
```

Comment:
We need "login_and_team_required" decorators for these views?

Reply:
+1 for adding auth and permission decorators
```suggestion
@login_and_team_required
@permission_required("events.add_timeouttrigger")
def create_timeout_event_view(request, team_slug: str, experiment_id: str):
```